### ISSUE TITLE
Confusing hint for persisting order of lists/cards

### ISSUE THREAD

**Issue by davidrunger** [appacademy/TrelloClone#29]
In Phase IV, bullet 1, we point the students to [this SO answer](http://stackoverflow.com/questions/15633341/jquery-ui-sortable-then-write-order-into-a-database/15635201#15635201), which mentions using a certain [serialize method](http://api.jqueryui.com/sortable/#method-serialize) in order to persist the order of sortable elements to the DB. However, our solution does not use this method, and takes basically a completely different approach. Furthermore, I don't really see a particularly elegant way to use this proposed serialize method to both update the Backbone models and the DB records to reflect the new order of sortable items after dragging and dropping.

In summary, either I am missing something, or we are misleading students and directing them down an un-fruitful path by suggesting that they use the above-referenced serialize method.
